zapai/encoder.go

```diff
@@ -86,8 +87,23 @@ func (g *gobber) decode(b []byte) (*appinsights.TraceTelemetry, error) {
 
 // fieldStringer evaluates a zapcore.Field in to a best-effort string.
 func fieldStringer(f *zapcore.Field) string {
-	if f.Type == zapcore.StringType {
+	switch f.Type {
+	case zapcore.StringType:
 		return f.String
+
+	case zapcore.Int64Type:
+		return strconv.FormatInt(f.Integer, 10)
+
+	case zapcore.Uint16Type:
+		return strconv.FormatInt(f.Integer, 10)
+
+	case zapcore.ErrorType:
+		return f.Interface.(error).Error()
+
+	case zapcore.BoolType:
+		return strconv.FormatBool(f.Integer == 1)
+
```

Comment:
why do you have all this extra whitespace? 🥲

Reply:
you mean the empty line right? removed.